woocommerce/woocommerce-android
# Product creation using package photom: Media picker

This PR adds a product picker dialog implemented in Compose and integrates the media picker library for selecting an image. Supported media sources are: **Local device media**, **device camera** & **WP media library**.

[Demo](https://github.com/woocommerce/woocommerce-android/assets/1522856/cf9f813e-58f8-4ad3-b831-a45351178ed9)

**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Tap on Use package photo button
5. Notice a media picker dialog with 3 options appears
6. Try each one and verify an image is properly displayed in the package photo bottom sheet
7. Once a photo is loaded, tap on the Pencil button above the image
8. Try changing the image and verify it works
